### ISSUE TITLE
How activate the IdentityServer detailed logging

### ISSUE THREAD

**Issue by paulvanbladel** [joeaudette/cloudscribe#344]
I'm currently trying to configure a Client Api, but running into trouble eg. where I specify "Implicit" as grant type rather than "implicit" with lower case and were the grant type is not well digested.
I would like to have the same logging as the out of the box identity server style:
```
16:48:00 Information] IdentityServer4.Endpoints.AuthorizeEndpoint
ValidatedAuthorizeRequest
{
  "ClientId": "js",
  "ClientName": "JS Client",
  "RedirectUri": "http://localhost:5003/callback.html",
  "AllowedRedirectUris": [
    "http://localhost:5003/index.html",
    "http://localhost:5003/callback.html",
    "http://localhost:5003/silent.html",
    "http://localhost:5003/popup.html"
  ],
  "SubjectId": "anonymous",
  "ResponseType": "id_token token",
  "ResponseMode": "fragment",
  "GrantType": "implicit",
  "RequestedScopes": "openid profile api1",
  "State": "2879d592141c4679988f5fb5be1d3079",
  "Nonce": "b0c4521eb02846e2883f458a7c052fc8",
  "Raw": {
    "client_id": "js",
    "redirect_uri": "http://localhost:5003/callback.html",
    "response_type": "id_token token",
    "scope": "openid profile api1",
    "state": "2879d592141c4679988f5fb5be1d3079",
    "nonce": "b0c4521eb02846e2883f458a7c052fc8"
```
Identityserver uses serilog and following code block activates this logging.
```
public static IWebHost BuildWebHost(string[] args)
        {
            Log.Logger = new LoggerConfiguration()
                .MinimumLevel.Debug()
                .MinimumLevel.Override("Microsoft", LogEventLevel.Warning)
                .MinimumLevel.Override("System", LogEventLevel.Warning)
                .MinimumLevel.Override("Microsoft.AspNetCore.Authentication", LogEventLevel.Information)
                .Enrich.FromLogContext()
                .WriteTo.File(@"identityserver4_log.txt")
                .WriteTo.Console(outputTemplate: "[{Timestamp:HH:mm:ss} {Level}] {SourceContext}{NewLine}{Message:lj}{NewLine}{Exception}{NewLine}", theme: AnsiConsoleTheme.Literate)
                .CreateLogger();

            return WebHost.CreateDefaultBuilder(args)
                    .UseStartup<Startup>()
                    .ConfigureLogging(builder =>
                    {
                        builder.ClearProviders();
                        builder.AddSerilog();
                    })
                    .Build();
        }
```
So, especially, the MinimumLevel.Override statements are important here.
How can I do the samen in Cloudscribe.

Thanks a lot
cheers
paul.

**Comment by joeaudette** [comment]
I don't ship apps, I only ship nuget packages. You control the app code and can update the logging configuration in Program.cs of your app

**Comment by paulvanbladel** [comment]
Sorry Joe, I was referring to the sample app.

**Comment by joeaudette** [comment]
you are free to copy and modify the sample however you wish.

**Comment by paulvanbladel** [comment]
Joe,
Yeah, definitely nothing wrong with the logging.
The reason why I didn't see any identity server related logging was due to cors issue.
I generated the cloudscribe server with the .net core CLI template.
In the startup.cs of the template there still cors configured for a test server.
Obviously CORS caused that I didn't see any logging :)

https://github.com/joeaudette/cloudscribe.templates/blob/master/Content/WebApp/Startup.cs

```
options.AddPolicy("default", policy =>
                {
                    policy.WithOrigins("http://localhost:55347", "http://localhost:55347")
                        .AllowAnyHeader()
                        .AllowAnyMethod();
                });

```
Maybe in template it's better to put a comment that the user can configure cors and keep the policy for debugging open with policy.AllowAnyOrigin().
I would be happy to provide a PR for this, if you agree.
Cheers
paul.
